plugin/src/lib.rs

```diff
@@ -496,6 +496,28 @@ impl<'a, 'cx> Folder for MutatorPlugin<'a, 'cx> {
                     attrs,
                 })
             }
+            Expr {
+                id,
+                node: ExprKind::WhileLet(pat, cond, block, opt_label),
+                span,
+                attrs,
+            } => {
+                let (n, current, sym, flag, mask) = self.add_mutations(
+                        cond.span,
+                        &["replacing whileLet Some() condition"],
+                    );
+               
+                let mut_block = quote_expr!(self.cx(), {
```

Comment:
I have used `quote_expr!`. is it correct?

Reply:
It's correct